src/compojure/core.clj

```diff
@@ -134,9 +134,17 @@
 (defn- wrap-route-info [handler route-info]
   (fn
     ([request]
-     (handler (assoc request :compojure/route route-info)))
+     (let [full-route (str (:compojure/context-path request) (second route-info))]
```

Comment:
I think in another PR I suggested `:compojure/route-context`. Also better to use destructuring outside the fn rather than `second`.